public/static/docs/command-reference/diff.md

```diff
@@ -46,148 +57,142 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Examples
 
-For these examples we can use the chapters in our
-[Get Started](/doc/get-started) section, up to
-[Add Files](/doc/get-started/add-files).
+For these examples we can use the [Get Started](/doc/get-started) repository.
 
 <details>
 
-### Click and expand to setup example
+### Click and expand to setup the examples project
```

Comment:
Maybe just `...the example` like in line 105?

p.s. the `import-url` cmd ref could also use this update 🙂

Reply:
don't want to go with other pages... I hope all these setup instructions will go away soon